.vib/drupal/goss/vars.yaml

```diff
@@ -0,0 +1,24 @@
+binaries:
+  - apachectl
+  - gosu
+  - mysql
+  - php
+  - render-template
+  - drush
+files:
+  - paths:
+    - /opt/bitnami/drupal/sites/default/settings.php
+    - /opt/bitnami/drupal/sites/default/files/.htaccess
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/drupal
+      - /opt/bitnami/drupal
+      - /opt/bitnami/drupal/sites/default/files
+      - /opt/bitnami/drupal/themes
+      - /opt/bitnami/drupal/modules
```

Comment:
We are not actually setting the permissions for these 2 dirs, I would move them to another element of the `.Vars.directories` array without the `mode` variable.

Reply:
Thanks for the suggestion! I applied this change 👍